preprocessing/create_fashion_gen_metadata.ipynb

```diff
@@ -0,0 +1,1288 @@
+{
```

Comment:
Do you know if there is any way to see the code in this colab directly? it's quite hard to review in this format :)

Reply:
Also, make sure to add the header comment that you have in the other python files here too about the license, etc.